### ISSUE TITLE
Are blank tiles needed?

### ISSUE THREAD

**Issue by JohnDMcMaster** [whitequark/groupXIV#2]
sipr0n tiles are generated only for the visible image. However, groupXIV seems to center images in padded squares, inserting many blank tiles. Can you provide some background why you went this direction? This adds a lot of unnecessary files, especially for high aspect ratio images.

**Comment by whitequark** [comment]
I can. I tried to implement it in a better way and for some reason my ImageMagick code didn't work. Since most of my images were nearly square anyway, I didn't really care.

I remember that working on the tile cutter was pretty frustrating so I'd really appreciate if you took a look at it yourself first. But if that doesn't work I'll dig in.

**Comment by JohnDMcMaster** [comment]
Gotcha

This is purely a rendering issue for me since I'm using my existing tile cutter right now (https://github.com/SiliconAnalysis/pr0nmap). Does this mean that you could adjust the js pretty easily if you didn't have to worry about the cutter?

Also note sometimes I stitch directly to files since I have panos that exceed allowed .jpg size.

**Comment by whitequark** [comment]
JS already doesn't care about missing tiles though, is there any specific problem you can point me to?

**Comment by JohnDMcMaster** [comment]
Yes. It causes the field of view to be limited to the tile cutter center area. See for example: https://siliconpr0n.org/map/tmp/pr0nmap2/

Seems to get stuck in the bottom center area as if you were zooming into a padded square layout.

**Comment by JohnDMcMaster** [comment]
Poke?
